libs/shared/ui/src/lib/hero-banner-util-button/hero-banner-util-button.component.html

```diff
@@ -0,0 +1,21 @@
+<biosimulations-hover-open-menu>
+   <biosimulations-topbar-menu-item heading="Utilities" icon="construction" id="trigger">
+   </biosimulations-topbar-menu-item>
+   <ng-container id="items">
```

Comment:
so this new component allows local navigation to the utilities using local routing, but most apps don't have these routes.  So the new design is to (1) navigate to the right app, and (2) then use the utilities of that app?  This could make sense, but maybe it deserves some discussion.

Reply:
This new component composition is intended to reduce and prevent code duplication. Notice that each app has a reference to a `utils` submodule. Each relative ui `app.component` already had a reference to `utils` as a navigation button linked with `routerLink`. Given the fact that these `biosimulations-dropdown-menu-item`s are simply re-iterated in each of the ui apps, this is logical solution to prevent the duplication. Using `routerLink` to reference each of the `utils` items in this implementation works as identically as the original implementation, but without the code duplication.

Reply:
I'm surprised that each app has local implementations of the same utils, is that really the case?